cli/crates/cli/src/start.rs

```diff
@@ -85,7 +85,11 @@ async fn log_reporter(mut message_receiver: UnboundedReceiver<ServerMessage>, lo
     while let Some(message) = message_receiver.recv().await {
         #[allow(clippy::single_match)] // will certainly change in the future
         match message {
-            ServerMessage::Ready { listen_address, port } => {
+            ServerMessage::Ready {
+                listen_address,
+                port,
+                is_federated: _,
```

Comment:
I am wondering what happens here if federated is true. Engine v1 can't serve a federated graph, so maybe an error / warning would make sense?

Reply:
@tomhoule WDYM? I changed the prod server to also support this

Reply:
Aah nice, I hadn't caught on to that, it all makes sense.